<launch> 
    <!-- false此项可提升性能 -->
    <arg name="isVis" default="false"/>
    <arg name="isSave" default="false"/>


    <node pkg="event_traj_detector" type="event_traj_detector_node" name="event_traj_detector_node" output="screen">
        <rosparam command="load" file="$(find event_traj_detector)/launch/config/detector_default.yaml" />

        <param name="isVis" type="bool" value="$(arg isVis)"/>
        <param name="isSave" type="bool" value="$(arg isSave)"/>
    </node>         


</launch>